### PR TITLE
glab: add new versions

### PR DESCRIPTION
**Pull request by mpokorny**

### CODE REVIEW

**Comment by alecbcs** [comment]
Hey @mpokorny! Thanks for the PR! Do you need all of these versions in Spack or would it be sufficient for your use case to simply add the latest version? (v1.30.0)

**Comment by mpokorny** [comment]
Simply adding the latest version would be sufficient for my use (changelog feature).  Removing intervening versions is fine by me.

**Review by tldahlgren — DISMISSED** [review]
Confirmed all of the new sha256, will defer merge decision to @alecbcs

File: var/spack/repos/builtin/packages/glab/package.py [hunk 1]
```diff
@@ -14,6 +14,12 @@ class Glab(Package):
 
     maintainers("alecbcs")
 
+    version("1.30.0", sha256="d3c1a9ba723d94a0be10fc343717cf7b61732644f5c42922f1c8d81047164b99")
+    version("1.29.4", sha256="f6c628d376ea2db9872b1df20abc886281ba58b7bdf29f19dc179c541958640b")
+    version("1.29.3", sha256="4acb68e831948ffa8c31bc777a3100f2459c6360edaab65e1720805dbc04b8b5")
+    version("1.29.2", sha256="33eea103d4be15e349d56707962e9a1e0adb21454bcf261680de55b063a429e5")
+    version("1.29.1", sha256="780bd9cd8c5dac9848bde5210faf6384cf752116853de23cefb6dcfa75e4dc5d")
+    version("1.29.0", sha256="5ea8c805f3555352c2cc55cf174f1430dffe3a19570ce25b1889a3903fd0dd0f")
```
Comment (alecbcs) [inline var/spack/repos/builtin/packages/glab/package.py]:
```suggestion
    version("1.30.0", sha256="d3c1a9ba723d94a0be10fc343717cf7b61732644f5c42922f1c8d81047164b99")
```

Suggestion to drop the previously added non-latest versions. @mpokorny if this looks good to you you can commit the suggestion and we'll merge into develop.

**Review by alecbcs — APPROVED** [review]
Confirmed SHA. Looks good to me! Thank you @mpokorny 🎉

**Pull request merged**